octoprint_octorelay/driver.py

```diff
@@ -12,6 +12,8 @@ def __init__(self, pin: int, inverted: bool, pin_factory=None):
         self.pin = pin # GPIO pin
         self.inverted = inverted # marks the relay as normally closed
         self.handle = LED(pin, pin_factory=pin_factory, initial_value=inverted)
+        # release immediately, avoid lock, allow physical buttons to operate same relays:
+        self.handle.pin_factory.release_pins(self.handle, self.pin)
```

Comment:
@ch3p4ll3 , do you know if it's possible somehow to avoid reservation at all?
I could not find any useful information about that problem, except the comment in `Factory` code:
https://gpiozero.readthedocs.io/en/latest/_modules/gpiozero/pins.html#Factory

```
Descendents *may* override the following methods, if applicable:
* :meth:`reserve_pins`
```

So that theoretically, a custom Pin Factory could have no-op `reserve_pins()` method.
But the problem is that we're using `None` for `pin_factory` relying on internal election between `lgpio` and `RPI.GPIO`.

I tested this line on my Pi 4, and it works, restoring previous operation for the same relay by a Physical Button plugin (it uses RPI.GPIO), though, the line itself does not look much pretty to me.

Reply:
Election's happening in `_default_pin_factory()` called by public `ensure_pin_factory` of the `Device` class
https://gpiozero.readthedocs.io/en/latest/_modules/gpiozero/devices.html#Device

Reply:
Or maybe we could replace `LED` with a composite device, that
1. inits pin factory,
2. extends it somehow or overrides its `reserve_pins` method with no-op,
3. creates actual `LED` instance
That's what I'm thinking about.
Though, perhaps it's not much cleaner than the line that releases afterwards

Reply:
Ah... it's actually static

```py
@staticmethod
    def ensure_pin_factory():
```

so that we could probably call `Device.ensure_pin_factory()` without instantiating it!

Reply:
If I make a build, could you check that it works on Raspberry Pi 5 (general startup, relay switching), @ch3p4ll3 ?

Reply:
> If I make a build, could you check that it works on Raspberry Pi 5 (general startup, relay switching), @ch3p4ll3 ?

Yep, no problem